axelor-production/src/main/resources/views/ManufOrder.xml

```diff
@@ -252,6 +252,7 @@
   	  	<view-param name="popup-save" value="false"/>
 	  	<view-param name="forceEdit" value="true" />
 	  	<context name="_showRecord" expr="eval: __this__.id" />
+	  	<context name="_statusSelect" expr="eval: __this__.statusSelect" />
```

Comment:
what is the need to set '_statusSelect' context variable , its not working directly with 'statusSelect' ?